packages/library/src/core/component.ts

```diff
@@ -197,6 +195,10 @@ export class Component extends BaseComponent {
     this.status = Status.locked
   }
 
+  get global() {
+    return this.#controller.global
+  }
```

Comment:
I'm split on this -- I thought about it for a very long time, and I definitely see the value as a shortcut, on the other hand I remember the pre-port state of `lab.js` where a lot of global state was attached to the component, and think about whether to make it explicit. I don't have a strong opinion either way, but I think we should discuss briefly (a) if there's a way to make this private (I can't find one -- there doesn't seem to be support in either ES or TS for private accessors or (b) to expose this with read/write capabilities. Would love to hear what you think!

Reply:
Our refactor depends heavily on being able to access this global state from any component. I would advocate for keeping it public, although I think `readonly` would make sense with the caveat that we do need to be able to mutate the state during the top-level sequence's `prepare` event